### PR TITLE
[#10253] Bottom system panel is white on dark mode on Android

### LINKED ISSUE #10253: Bottom system panel is white on dark mode on Android
# Bug Report

## Problem

photo_2020-03-31 13 17 10

#### Expected behavior
photo_2020-03-31 13 18 18

### Additional Information

[comment]: # (Please do your best to fill this out.)
- Status version: nightly 31/03/2020
- Operating System: Android (Xiaomi Mi Note9), dark mode is selected in phone settings

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #10253

### CODE REVIEW

File: android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
         <item name="android:forceDarkAllowed">true</item>
         <item name="android:windowTranslucentStatus">false</item>
         <item name="android:statusBarColor">?attr/backgroundColor</item>
+        <item name="android:navigationBarColor">?attr/backgroundColor</item>
```
Comment (Ferossgp) [inline android/app/src/main/res/values/styles.xml]: I think this match only the system theme, and will not follow the preferred one in the app. We need something like we do with the status-bar, to control it from the app. Also now the bg and buttons are both white on my phone
photo_2020-06-18 13 23 06
photo_2020-06-18 13 23 07

Comment (flexsurfer) [reply android/app/src/main/res/values/styles.xml]: > I think this match only the system theme, and will not follow the preferred one in the app.

that's fine. 

> Also now the bg and buttons are both white on my phone

that's not cool :(

**Comment by flexsurfer** [comment]
@Ferossgp could you please try now if it works for you, thanks

**Review by Ferossgp — APPROVED** [review]
LGTM :+1:

**Comment by churik** [comment]
Works great with light and dark theme on Android 10

**Pull request merged**